utils/order-utils/src/signatures.js

```diff
@@ -91,14 +95,32 @@ module.exports = {
       s,
     }
   },
-  getEmptySignature() {
+  getEmptySignature(verifyingContract) {
     return {
       signatory: EMPTY_ADDRESS,
-      validator: EMPTY_ADDRESS,
+      validator: verifyingContract,
       version: signatures.INTENDED_VALIDATOR,
       v: '0',
       r: '0x0',
       s: '0x0',
     }
   },
+  async isSignatureValid(order) {
```

Comment:
I don't believe the `async` is needed here since there is no `await` in the function body

Reply:
Removed this as the corresponding await